routes/users.js

```diff
@@ -0,0 +1,51 @@
+const express = require('express');
+const bcrypt = require('bcryptjs');
+const router = express.Router();
+
+const userDB = require('../data/helpers/usersDb');
+
+router.post('/register', (req, res) => {
+    const user = req.body;
+    user.password = bcrypt.hashSync(user.password, 15);
+
+    userDB.add(user)
+        .then(id => {
+            res.status(201).json(id);
+        })
+        .catch(err => {
+            res.status(500).json({ errorMessage: 'Failed to insert user' });
+        }) 
+});
+
+router.post('/login', (req, res) => {
+   const user = req.body;
+   
+   userDB.get(user)
+    .then(users => {
+        if(users.length && bcrypt.compareSync(user.password, users[0].password)) {
```

Comment:
Like the usage of boolean `&&` operator here to check the users' array and the password within a single if statement 🔥